src/Components/AppShell/TopNavbar.js

```diff
@@ -111,8 +116,8 @@ class TopNavbar extends Component {
     const gasLimit = this.props.core.gasLimit
     const snapshots = this.props.core.snapshots
     const isMining = this.props.core.isMining
+    const isLogsPage = this.props.location.pathname === '/logs'
```

Comment:
If you press the LOGS link in the nav bar when you're already in the logs page, the button disappears